src/Server.cpp

```diff
@@ -597,7 +597,7 @@ void cServer::PrintHelp(const AStringVector & a_Split, cCommandOutputCallback &
 	for (AStringPairs::const_iterator itr = Callback.m_Commands.begin(), end = Callback.m_Commands.end(); itr != end; ++itr)
 	{
 		const AStringPair & cmd = *itr;
-		a_Output.Out(Printf("%-*s%s\n", static_cast<int>(Callback.m_MaxLen), cmd.first.c_str(), cmd.second.c_str()));
+		a_Output.Out(Printf("%-*s%s\n", static_cast<int>(Callback.m_MaxLen), cmd.first.c_str(), (" - " + cmd.second).c_str()));
```

Comment:
Okay... silly me. I don't know much about format strings.  Is there any documentation you can give me?